include/solvers/mpm_explicit.tcc

```diff
@@ -281,9 +281,8 @@ bool mpm::MPMExplicit<Tdim>::solve() {
 #ifdef USE_MPI
 #ifdef USE_GRAPH_PARTITIONING
     // Run load balancer at a specified frequency
-    if (step_ % nload_balance_steps_ == 0 && step_ != 0)
-      this->mpi_domain_decompose(false);
-
+    // if (step_ % nload_balance_steps_ == 0 && step_ != 0)
+    //   this->mpi_domain_decompose(false);
```

Comment:
Do you do this intentionally?

Reply:
It's causing an issue at large deformations, so temporarily disabling to avoid this issue and keeps `develop` functioning. I have described it in PR.

Reply:
Ok! No problem. @kks32 For the load balancer, I have an idea to improve the performance. Will open an issue.